tests/fixtures/check_for_cran.yml

```diff
@@ -0,0 +1,186 @@
+        R-release (experimental)\",\n\t\"cran-name\": \"r-release-osx-x86_64\",\n\t\"rversion\":
+        \"r-release\",\n\t\"os-type\": \"macOS\",\n\t\"macos-version\": \"elcapitan\",\n\t\"cpu-type\":
+        \"x86_64\",\n\t\"os-info\": \"Mac OS X 10.11.6 15G1217\",\n\t\"compilers\":
+        \"Apple LLVM version 8.0 (clang-800.0.42.1); GNU Fortran 4.2.3\",\n\t\"docker-image\":
+        null,\n\t\"sysreqs-platform\": \"osx-x86_64-clang\",\n\t\"categories\": [\"macOS\"],\n\t\"node-labels\":
+        [\"macos\", \"elcapitan\", \"r-release\"],\n\t\"queue\": \"elcapitan\"\n    },\n\n
+        \   {\n\t\"name\": \"ubuntu-rchk\",\n\t\"description\": \"Ubuntu Linux 16.04
+        LTS, R-devel with rchk\",\n\t\"cran-name\": null,\n\t\"rversion\": \"r-devel\",\n\t\"os-type\":
+        \"Linux\",\n\t\"cpu-type\": \"x86_64\",\n\t\"os-info\": \"Ubuntu 16.04 LTS\",\n\t\"compilers\":
+        \"clang 3.8.0-2ubuntu4\",\n\t\"docker-image\": \"ubuntu-rchk\",\n\t\"sysreqs-platform\":
+        \"linux-x86_64-ubuntu-gcc\",\n        \"output-parser\": \"rchk\",\n\t\"categories\":
+        [\"Checks for compiled code\"],\n\t\"node-labels\": [\"linux\"],\n\t\"queue\":
+        \"linux\"\n    },\n\n    {\n\t\"name\": \"solaris-x86-patched\",\n\t\"description\":
+        \"Oracle Solaris 10, x86, 32 bit, R-patched (experimental)\",\n\t\"cran-name\":
+        \"r-patched-solaris-x86\",\n\t\"rversion\": \"r-patched\",\n\t\"os-type\":
+        \"Solaris\",\n\t\"cpu-type\": \"x86_64\",\n\t\"os-info\": \"SunOS 5.10 Generic_147148-26
+        i86pc i386 i86pc\",\n\t\"compilers\": \"GCC 5.2.0\",\n\t\"docker-image\":
+        null,\n\t\"sysreqs-platform\": \"solaris-10\",\n\t\"categories\": [\"Solaris\"],\n\t\"node-labels\":
+        [\"solaris\"],\n\t\"queue\": \"solaris\"\n    }\n]\n"
+  recorded_at: 2020-02-04 14:44:17 GMT
+  recorded_with: vcr/0.4.1.93, webmockr/0.5.1.98
+- request:
+    method: post
+    uri: https://builder-test.r-hub.io/api/check/submit
+    body:
+      encoding: ''
+      string: '{"email":"<<<my_email>>>","token":"<<<my_rhub_token>>>","package":"myminimalpackage","version":"1.0.0","platform":["windows-x86_64-devel","ubuntu-gcc-release","fedora-clang-devel"],"env":{"_R_CHECK_FORCE_SUGGESTS_":["true"],"_R_CHECK_CRAN_INCOMING_USE_ASPELL_":["true"]},"check_args":"--as-cran","file":"H4sIAAAAAAAAA+2V0WrbMBSGfa2nOPgqgdiRXNkZXjsSkmy0JFlIs92MXWi2nGmzpSDLhWz0\n3SvbuSiBUQYhZcMfGPnYxzo6+vXj4lAIKQqW71nyk+34cDa/n25u19vbjyvnTGBLRGk9klGI\nn48NURA65Coi0RUmdBQ5mIQ0iBzAzgWoSsO0XUrBeJ7zP+e99P4fZd3KHkNxchDQVpjcPm8G\nmLKSo89cl0LJGIiPfYwmlfmudDnexIDAkvT2NkHJ3k48cAk34N4pdwAZK0R+qMOZ4jbmBRN5\nHf5QqeLjVBU+E9IdNHMAaGXL3djJXFYZm+4mmrv9fh8tbZaxF9cxZEKXBkqeKJnCdRP5bTTO\nlPK/Mf0OzXiZaLE3zYqXbW9wbA4ypcHw0gi582FZ5Ubsc+6jhUi4LG3XH9YLL0AL9uswY4bF\nYHTF0VwmKrVfxPBp+957g1acp+VUFXuRs7aMVOi4oWkMAQ6whwMPUyA0pjQm1H44fQvtUTru\nXwx3CuzOwBfb7wBst1/RBfU/lX24mizn9+vJdH6+Gi/53/LM/2HtfxKQzv+vov9mePYatcCj\nMPwL/SkhoQNhp/9r6H+88zcX839AT/0fBaPO/5cgg2sPskom9Q+s14ff8Iicjo6Ojo7/nie0\nUaa5ABAAAA=="}'
```

Comment:
the "file" part seems to change a bit from time to time although the same package was created. I'm not sure how to best handle this?